scripts/generate-manifest.js

```diff
@@ -83,6 +83,7 @@ function generateVectorManifest(sources, opts) {
           layers.push(manifestLayerV2(source, opts.hostname));
           break;
         case 6:
+        case 7: // v6 and v7 manifest schema are the same
```

Comment:
oh man, the intricacies of cascading a switch/case statement :cold_sweat: ;) great idea of adding that comment